docs/FRSKY_R9.md

```diff
@@ -21,31 +21,49 @@ The Frsky R9M transmitter module and R9MX receiver are commercially available an
   </tbody>
 </table>
 
-Comment: mLRS also supports the R9MM receiver. However, flashing the R9MM with ST-Link is really tedious and requires top soldering skills or employing other tricks, as one needs to connect to four tiny solder pads. You really should consider using the R9MX receiver instead. Note: The R9MM has a different wiring diagram than the R9MX.
+Note that R9 uses a slightly older LORA chip family, which doesn't support the 31 Hz mode. It also happens to be incompatible with the newer LORA chips when the specific spreading factor used by the mLRS 19 Hz mode is selected.  Unfortunately, this means that R9 hardware can't connect with the other mLRS boards which support 868/915 MHz.
 
 ## R9M Tx Module ##
 
-The R9M module is unfortunately limited with respect to serial ports. It provides acess to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
+The R9M module is somewhat limited with respect to serial ports. It provides access to only one serial port, which moreover has inverted TTL signals. In order to use this serial port, these conditions apply:
 
-1. One needs to use a "Frsky inverter" dongle. You either can buy one or build it yourself.
+1. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to re-power the module to make any change effective.
 
-2. The serial port of the R9M module can be configured to work either as "serial" or "CLI". This is done by setting the dip switch 1 (left dip switch): off (switch down) = CLI, on (switch up) = serial. Note that the dip switch position is read only at power up, i.e., one needs to repower the module to make any change effective.
+2. One needs to use a "Frsky inverter" dongle to connect a standard serial adapter. You either can buy one or build it yourself.
+
+3. If you use the mLRS Lua configuration script for configuration and thus don't need the CLI for configuration, you can avoid the inverter dongle and use the one of the supported ESP32 boards with the mlrs-wifi-bridge sketch to connect via MAVLink wirelessly to a Ground Control Station.  The ESP32 can directly use the inverted serial signals.
```

Comment:
again, the one has nothing to do with the other.
The lua script only means that one doesn't have to fool around with the dip switch, but doesn't free one from the inverted signals on the serial. 

maybe if you just kill the frist half sentence, and start with "You can avoid the inverter dongle ..."

if you really want to mention the lua script, it IMHO rather should be appened to point 1. Saying that if teh Lua is used one doesn't need to deal with the dip.

Reply:
I think we talked about the possibility, but didn't yet provide serial passthrough (inverter) in the sketch to allow the ESP32 to function as a simple inverter.  Further, as far as I know, there is no easy way to access the CLI via the WiFi bridge.  This means that even if the ESP32 and sketch is used, we also need a dedicated inverter to access the CLI.  Unless, of course, we don't need the CLI and use the LUA script instead.  Correct me if I'm wrong on any of this.  I want new users to fully  understand if/why they might need an inverter, especially since it might not be easy for them to find or build one.

Reply:
we discussed some future options, but I'm not refereing to any since ... well ... they are not existing :) I'm talking only about current status.
In order to use the serial port, one needs to do some inversion. That fact is independent on usage, or even lua or not.
The usage of the lua is in no way related to using a wifi dongle (all it means is that one can keep the wifi dongle installed permanently, but that's bnot what is being said)
That's kind of all I'm saying :)